### PR TITLE
Leads section improvements. 

### LINKED ISSUE #111: Welcome the organizer of a GDG with a special message
When the user is one of the organizers of the user's home GDG set a flag and a message to congratulate for the great work.

In the future, the gdg manager will have an extra menu for additional actions.

### PR DESCRIPTION
**Pull request by tasomaniac**
Don't merge this yet. 

I couldn't test most of this pull request because I cannot login to the app. I only send this for someone to review. 

Fixes #111 when merged.

### CODE REVIEW

File: app/src/main/java/org/gdg/frisbee/android/activity/MainActivity.java [hunk 1]
```diff
@@ -175,6 +176,29 @@ protected void onResume() {
...
+        checkOrganizer();
+    }
+
+    @Override
+    public void onConnected(Bundle bundle) {
+        super.onConnected(bundle);
+        checkOrganizer();
+    }
+
+    private void checkOrganizer() {
```
Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/activity/MainActivity.java]: just the naming, seems too much organizers and checks.

the method name could be "updatePages"
OrganizerChecker.OrganizerResponseHandler should be just OrganizerChecker.ResponseHandler

File: app/src/main/java/org/gdg/frisbee/android/app/OrganizerChecker.java [hunk 2]
```diff
@@ -37,7 +39,9 @@ public boolean isOrganizer() {
     }
 
     public void checkOrganizer(GoogleApiClient apiClient, final OrganizerResponseHandler responseHandler) {
-        final String currentId = Plus.PeopleApi.getCurrentPerson(apiClient).getId();
+        final Person plusPerson = PrefUtils.isSignedIn(apiClient.getContext())
```
Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/app/OrganizerChecker.java]: maybe extract this into a method getCurrentPersonId that is nullable

Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/app/OrganizerChecker.java]: makes sense.

File: app/src/main/java/org/gdg/frisbee/android/activity/MainActivity.java (hunk 1 above)
Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/activity/MainActivity.java]: Exactly. I can make that change. 
And what about `OrganizerChecker.Callbacks`

Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/activity/MainActivity.java]: Yes, good!


  [image: --]
Friedger Müffke
[image: http://]about.me/friedger
<http://about.me/friedger>


2015-05-05 22:06 GMT+02:00 Said Tahsin Dane <notifications@github.com>:

**Comment by tasomaniac** [comment]
Pull Request updated :)

**Pull request merged**